pkg/env/arcsjs-apps/pkg/arcs-import.js

```diff
@@ -0,0 +1 @@
+export * from './env/arcs-local.js';
```

Comment:
Needs header here and next couple of files.

Reply:
There is no sane universe in which `export * from path` is copyrightable, but ok. :)